submissions/VikTorNik/html-css-popup/style.scss

```diff
@@ -0,0 +1,245 @@
+* {
+    padding: 0;
+    margin: 0;
+    border: 0;
+}
+
+*,
+*:before,
+*:after {
+    box-sizing: border-box;
+}
+
+:focus {
+    outline: none;
+}
+
+$color-menu-dark: rgb(7, 68, 64);
+$color-menu-focus: rgb(189, 226, 234);
+$color-submenu: rgb(5, 37, 41);
+$color-submenu-fon: rgb(184, 242, 242);
+$color-submenu-focus: rgba(255, 255, 255, 0.5);
+$color-header: rgb(68, 202, 206);
+$color-mein: rgb(90, 161, 177);
+
+html,
+body {
+    height: 100%;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 1.8rem;
+}
+
+a {
+    text-decoration: none;
+}
+
+.wrapper {
+    min-height: 100%;
+    display: flex;
+    flex-direction: column;
+}
+
+.header {
+    background-color: $color-header;
+}
+
+.nav {
+
+    .nav__menu {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-around;
+        height: 50px;
+
+        .li-menu,
+        .content-button,
+        .content-button-add {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            width: 100%;
+            font-weight: 500;
+            text-shadow: 3px 3px 8px $color-menu-dark;
+            text-transform: uppercase;
+            color: $color-submenu;
+
+            .item-menu:hover,
```

Comment:
Same story with too specific css selectors.

Reply:
Fixed it.